v2/emailpassword/pre-built-ui/setup/frontend.mdx

```diff
@@ -257,9 +257,7 @@ Before we initialize the `supertokens-auth-react` SDK let's see how we will use
 **Architecture**
```

Comment:
there is a section about "Redirect user to the login page" in https://deploy-preview-579--admiring-bhabha-7b1be9.netlify.app/docs/emailpassword/pre-built-ui/auth-redirection. Why can't users use `redirectToAuth` for angular and vue as well?